specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/preview/2020-07-20-preview1/examples/CancelSearchAsync.json

```diff
@@ -0,0 +1,10 @@
+{
+  "parameters": {
+    "api-version": "2020-07-20-preview1",
+    "endpoint": "https://resourcename.region.dev.communications.azure.net/",
```

Comment:
Let's remove the "region.dev" from the endpoints and switch to ".com". Let's do all of the other ones too.
```json
"endpoint": "https://resourcename.communications.azure.com/",
```

Reply:
Done.